tmt/cli.py

```diff
@@ -1844,15 +1844,15 @@ def clean_runs(
         context: Context,
         workdir_root: str,
         last: bool,
-        id_: Optional[str],
+        id_: tuple[str],
```

Comment:
The correct way is `tuple[str, ...]` - `tuple[str]` means a tuple with exactly one item, e.g. `("foo",)`.